opendevin/sandbox/plugins/jupyter/execute_cli

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
is this script getting used anywhere?

Reply:
To answer this & your previous question: "how does the agent send a python command to jupyter?"

`opendevin/sandbox/plugins` will be mounted to `/opendevin/plugins/` inside the sandbox container, and the `setup.sh` script will then add `/opendevin/plugins/jupyter` to `$PATH`, making sure that the sandbox `.execute` function has access to `execute_cli` tool.

Whenever the agent wants to execute Python, they can just do `echo "import math" | execute_cli` in bash and get the response in stdout (this part will likely be implemented by agent-specific logic, rather than prompting agents to do it.

Reply:
Oh nice! Maybe just add a note to this file that it's a convenience for the agent?

Should the setup script copy this file over to the sandbox environment?

Reply:
What I am doing now is mount the `opendevin/sandbox/plugins` folder directly into `/opendevin/plugins` inside the SSH sandbox (as read-only) so we did not perform any copy operation ;). I did not immediate come-up with how to perform `cp` operation once the image is built lol

Reply:
huh, do you mean we should just copy the `execute_cli` into some `..bin` folder, instead of adding the entire `/opendevin/plugins/jupyter` into `$PATH`? That could be another alternative too!

Reply:
Exactly! That way we don't have to mount additional directories. That'll make it easier to e.g. support jupyter on e2b

Reply:
done! i added `copy_to` method to the `Sandbox` interface, and supports how `Plugins` works through copying